src/easylink/pipeline_schema_constants/development.py

```diff
@@ -88,9 +88,30 @@
             )
         ],
         output_slots=[OutputSlot("step_3_main_output")],
+        nodes=[
+            BasicStep(
+                "step_3",
+                input_slots=[
+                    InputSlot(
+                        name="step_3_main_input",
+                        env_var="DUMMY_CONTAINER_MAIN_INPUT_FILE_PATHS",
+                        validator=validate_input_file_dummy,
+                    )
+                ],
+                output_slots=[OutputSlot("step_3_main_output")],
+            )
+        ],
+        edges=[
+            Edge(
+                in_node="step_3",
+                out_node="step_3",
+                output_slot="step_3_main_output",
+                input_slot="step_3_main_input",
+            )
+        ],
```

Comment:
Ok this took me a second. Maybe `edges` should be called `edges_to_next_iteration` or something?

Also, does this PR work with having a non-looping input to a looped step, say the original input as a secondary?